.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### Feature Requests ###
+Please read https://github.com/jquery/jquery/wiki/Adding-new-features ; most features should be implemented as plugins outside of jQuery.
+
```

Comment:
You might want to do like Node does and put this between comment (https://raw.githubusercontent.com/nodejs/node/master/.github/ISSUE_TEMPLATE.md) so this doesn't appear in the issue description.